public/js/app/components/MemoMenu.jsx

```diff
@@ -130,6 +131,10 @@ function MemoMenu(props) {
   // ** **************************** OTHER ******************************* */
 
   const { semesterList, roundList } = props
+  const convertedSemesterList = semesterList.map(({ term }) => ({
```

Comment:
You could be even more specific with the naming here, e.g. `semesterDropdownOptions` or something like that. Tells me as a reader more what it will be used as later on.

Reply:
I agree with and I changed it.